src/System.Management.Automation/FormatAndOutput/DefaultFormatters/PowerShellCore_format_ps1xml.cs

```diff
@@ -1151,13 +1151,7 @@ private static IEnumerable<FormatViewDefinition> ViewsOf_System_Management_Autom
 
                                             # replace newlines in message so it lines up correct
                                             $message = $message.Replace($newline, ' ').Replace(""`t"", ' ')
-                                            try {
-                                                $windowWidth = [Console]::WindowWidth
-                                            }
-                                            catch {
-                                                # fails if there is no console
-                                                $windowWidth = 120
-                                            }
+                                            $windowWidth = $Host.UI.RawUI.WindowSize.Width
```

Comment:
I guess RawUI is null is there is no console.

Reply:
Will add a check